server/modules/routing/readwritesplit/readwritesplit.cc

```diff
@@ -230,6 +230,18 @@ static bool rwsplit_process_router_options(Config& config,
                     success = false;
                 }
             }
+            else if (strcmp(options[i], "enable_causal_read") == 0)
```

Comment:
The options can probably be removed, they have been deprecated in MaxScale 2.2 and we will most likely remove all router options in 2.3.